code/modules/mob/living/carbon/human/species/monkey.dm

```diff
@@ -53,8 +54,8 @@
 	..()
 
 /datum/species/monkey/handle_dna(var/mob/living/carbon/human/H)
-	H.dna.SetSEState(MONKEYBLOCK,1)
-	genemutcheck(H, MONKEYBLOCK)
+	H.dna.SetSEState(MONKEYBLOCK,1,1)
```

Comment:
The `1` you added on the end here prevents the SE string from updating. You don't want to do that unless you're manually updating it later.